src/classes/com/sun/tdk/jcov/instrument/InstrumentationParams.java

```diff
@@ -102,6 +105,9 @@ public InstrumentationParams(boolean innerInvocations, boolean classesReload, bo
     }
 
     public InstrumentationParams(boolean innerInvocations, boolean classesReload, boolean dynamicCollect, boolean instrumentNative, boolean instrumentFields, boolean detectInternal, ABSTRACTMODE instrumentAbstract, String[] includes, String[] excludes, String[] callerIncludes, String[] callerExcludes, String[] m_includes, String[] m_excludes, InstrumentationMode mode, String[] saveBegin, String[] saveEnd) {
+        this(innerInvocations, classesReload, dynamicCollect, instrumentNative, instrumentFields, detectInternal, instrumentAbstract, includes, excludes, callerIncludes, callerExcludes, null, null, mode, saveBegin, saveEnd, null);
+    }
+    public InstrumentationParams(boolean innerInvocations, boolean classesReload, boolean dynamicCollect, boolean instrumentNative, boolean instrumentFields, boolean detectInternal, ABSTRACTMODE instrumentAbstract, String[] includes, String[] excludes, String[] callerIncludes, String[] callerExcludes, String[] m_includes, String[] m_excludes, InstrumentationMode mode, String[] saveBegin, String[] saveEnd, InstrumentationPlugin plugin) {
 
```

Comment:
It would be good to follow Java Code Conventions: Avoid lines longer than 80 chars.

Reply:
I would be happy to reformat my new added constructor and all the previously existing constructors. Will do.

However, the right thing to do would be to rework this class to use builder pattern. I have just created https://bugs.openjdk.java.net/browse/CODETOOLS-7902832 to that extent.